### PR TITLE
Rewording and fixes in documentation

### PR DESCRIPTION
**Pull request by CyrilleB79**
### Link to issue number:
None
### Summary of the issue:
Various rewording and fixes found while translating or checking the translation and the structure of the document.

### Description of user facing changes
Change log fixes and rewording.

### Description of development approach
* Fixing a list nesting issue in change log
* Various rewording and fixes in the change log
* Fixed a table format issue in user guide (but this issue had no consequence after html conversion anyway)
* Fixed lines with tailing spaces in the User Guide and in change log (but there was no consequence in HTML)

Note:
* Regarding tailing space fixing in the change log, I have done only the 2024.2 section because fixing them all would change too much lines for translators in old change log items.
* In the user guide, I have fixed them all because there are only few of them and they have been added recently.
* I can revert these fixes without any visible consequence if you prefer not to change the history. The idea for these fixes was to keep good practices.

### Testing strategy:
Check documentation on appVeyor
### Known issues with pull request:
None
### Code Review Checklist:

- [x] Documentation:
  - Change log entry
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] API is compatible with existing add-ons.
- [x] Security precautions taken.

### CODE REVIEW

File: user_docs/en/changes.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ There are many minor bug fixes for applications, such as Thunderbird, Adobe Read
 * Fixed a bug causing NVDA to fail to read the ribbon and options within Geekbench. (#16251, @mzanm)
 * Fixed a rare case when saving the configuration may fail to save all profiles. (#16343, @CyrilleB79)
 * In Firefox and Chromium-based browsers, NVDA will correctly enter focus mode when pressing enter when positioned within a presentational list (ul / ol) inside editable content. (#16325)
-* Column state change is automatically reported when selecting columns to display in Thunderbird message list. (#16323)
+* Column state change is now correctly reported when selecting columns to display in Thunderbird message list. (#16323)
```
Comment (CyrilleB79) [inline user_docs/en/changes.md:91]: Nothing automatic here.

@burmancomp, do you agree with this change?

Or maybe even the following would be better in English (@seanbudd, your opinion?):
```suggestion
* The column state change is now correctly reported when selecting the columns to display in Thunderbird message list. (#16323)
```

**Review by CyrilleB79 — COMMENTED** [review]
comments in the code to explain some changes.

**Comment by josephsl** [comment]
HI, there is – Windows 11 22H2 is also known as Windows 11 2022 Update. Same with 23H2 being called 2023 Update, so I would keep the entry as is. Thanks.

File: user_docs/en/changes.md [hunk 2]
```diff
@@ -32,38 +32,38 @@ There are many minor bug fixes for applications, such as Thunderbird, Adobe Read
...
+      * Desktop: `NVDA+control+pageUp` and `NVDA+control+pageDown`.
+      * Laptop: `NVDA+control+shift+pageUp` and `NVDA+control+shift+pageDown`.
   * Added a new unassigned input gesture to toggle the reporting of figures and captions. (#10826, #14349)
 * Braille:
   * Added support for the BrailleEdgeS2 and BrailleEdgeS3 displays. (#16033, #16279, @EdKweon)
   * A new braille mode called "display speech output" has been added. (#15898, @Emil-18)
     * When active, the braille display shows exactly what NVDA speaks.
     * It can be toggled by pressing `NVDA+alt+t`, or from the braille settings dialog.
-* Sound split: (#12985, @mltony)
+* A new sound split feature: (#12985, @mltony)
```
Comment (seanbudd) [inline user_docs/en/changes.md:46]: I don't think this change is necessary, brevity is fine here. This item is under "new features"

```suggestion
* Sound split: (#12985, @mltony)
```

File: user_docs/en/changes.md [hunk 3]
```diff
@@ -32,31 +32,31 @@ There are many minor bug fixes for applications, such as Thunderbird, Adobe Read
...
+* In Windows 11, NVDA will announce alerts from voice typing and suggested actions including the top suggestion when copying data such as phone numbers to the clipboard (Windows 11 22H2 Update and later). (#16009, @josephsl)
 * NVDA will keep the audio device awake after speech stops, in order to prevent the start of the next speech being clipped with some audio devices such as Bluetooth headphones. (#14386, @jcsteh, @mltony)
 * HP Secure Browser is now supported. (#16377)
 
 ### Changes
 
 * Add-on Store:
   * The minimum and the last tested NVDA version for an add-on are now displayed in the "other details" area. (#15776, @Nael-Sayegh)
-  * The community reviews action will be available, and the reviews webpage will be shown in the details panel, in all tabs of the store. (#16179, @nvdaes)
+  * The community reviews action will be available in all tabs of the store. (#16179, @nvdaes)
```
[the remark this answers is not in the document]
Comment (seanbudd) [reply user_docs/en/changes.md:59]: this looks good to me

File: user_docs/en/changes.md (hunk 1 above)
Comment (seanbudd) [reply user_docs/en/changes.md:91]: the original change you have proposed here is better

File: user_docs/en/changes.md [hunk 4]
```diff
@@ -32,31 +32,31 @@ There are many minor bug fixes for applications, such as Thunderbird, Adobe Read
...
     * It can be toggled by pressing `NVDA+alt+t`, or from the braille settings dialog.
-* Sound split: (#12985, @mltony)
+* A new sound split feature: (#12985, @mltony)
   * Allows splitting NVDA sounds into one channel (e.g. left) while sounds from all other applications are directed to the other channel (e.g. right).
   * Toggled by `NVDA+alt+s`.
 * Reporting row and column headers is now supported in contenteditable HTML elements. (#14113)
-* Added the option to disable the reporting of figures and captions in Document Formatting settings. (#10826, #14349)
-* In Windows 11, NVDA will announce alerts from voice typing and suggested actions including the top suggestion when copying data such as phone numbers to the clipboard (Windows 11 2022 Update and later). (#16009, @josephsl)
+* Added an option to disable the reporting of figures and captions in Document Formatting settings. (#10826, #14349)
+* In Windows 11, NVDA will announce alerts from voice typing and suggested actions including the top suggestion when copying data such as phone numbers to the clipboard (Windows 11 22H2 Update and later). (#16009, @josephsl)
```
[the remark this answers is not in the document]
Comment (seanbudd) [reply user_docs/en/changes.md:51]: These seem to be synonymous: https://betawiki.net/wiki/Windows_11_2022_Update

**Review by seanbudd — DISMISSED** [review]

File: user_docs/en/changes.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (seanbudd) [reply user_docs/en/changes.md:51]:
```suggestion
* In Windows 11, NVDA will announce alerts from voice typing and suggested actions including the top suggestion when copying data such as phone numbers to the clipboard (Windows 11 2022 Update and later). (#16009, @josephsl)
```

**Pull request merged**